lib/modules/apostrophe-doc-type-manager/public/js/chooser.js

```diff
@@ -324,6 +325,50 @@ apos.define('apostrophe-doc-type-manager-chooser', {
         position: { collision: 'flip' }
       });
     };
+
+    self.enableSortable = function () {
+      var sortableOptions = {
+        handle: '[data-apos-sort-item]',
+        update: function(event, ui) {
+          var $choices = self.$el.find('[data-chooser-choice]');
+          var proposedChoices = [];
+          var intact = true;
+
+          self.$choices.find('[data-chooser-choice]').each(function () {
+            var $choice = $(this);
+            proposedChoices.push({ value: $choice.attr('data-chooser-choice') });
+          });
+
+          // account for all original items
+          _.each(proposedChoices, function(proposed) {
+            var matched = false;
+            _.each(self.choices, function(choice) {
+              if (proposed.value === choice.value) {
+                matched = true;
+              }
+            });
+            if (matched === false) {
+              intact = false;
```

Comment:
Interesting. We don't sanity check regular up/down moves in this way. Do you feel it is necessary because you hit some cases where it didn't behave?

Reply:
I think if there are any elements with your attribute (which is new, after all) we can trust the DOM.